openff/interchange/_tests/unit_tests/components/test_packmol.py

```diff
@@ -225,18 +307,49 @@ def test_solvate_structure():
         pack_box(
             [Molecule.from_smiles("O")],
             [10],
-            box_size=([50] * 3) * unit.angstrom,
-            structure_to_solvate=benzene.to_topology(),
+            box_vectors=50 * numpy.identity(3) * unit.angstrom,
+            solute=benzene.to_topology(),
         )
 
     benzene.generate_conformers(n_conformers=1)
 
     topology = pack_box(
         [Molecule.from_smiles("O")],
         [10],
-        box_size=([50] * 3) * unit.angstrom,
-        structure_to_solvate=benzene.to_topology(),
+        box_vectors=50 * numpy.identity(3) * unit.angstrom,
+        solute=benzene.to_topology(),
     )
 
     assert topology.n_molecules == 11
     assert len([*topology.unique_molecules]) == 2
+
+
+def solvate_topology():
```

Comment:
The coverage report is complaining about `solvate_topology` not being covered, yet I recall writing a test to do it.

🤦